app/components/table-group.hbs

```diff
@@ -0,0 +1,43 @@
+{{#if (eq @models.length 0)}}
```

Comment:
I think `#if @models.length` can be used without an `eq` call, since `0 == false`